precompile/contracts/nativeminter/module.go

```diff
@@ -15,12 +15,13 @@ import (
 
 var _ contract.Configurator = &configurator{}
 
-// ConfigKey is the key used in json config files to specify this precompile config.
+// ConfigKey is the key used in json config files to specify this precompile precompileconfig.
```

Comment:
seems this didn't need to be changed? could we instead update the template to either `precompile config` or
`precompileconfig` and also make all the existing precompiles uniform?

Reply:
fixed